### DISCUSSION TITLE
Unable to find image 'gliderlabs/herokuish:latest-20.' locally

### DISCUSSION THREAD

**Discussion by constantgillet** [dokku/dokku#6352] [General]
Hello, 
I have changed nothing on my server but when I try building my app it shows:

`-----> Cleaning up...`
`-----> Building upgear from gliderlabs/herokuish:latest-20.`
`Unable to find image 'gliderlabs/herokuish:latest-20.' locally`
`docker: Error response from daemon: manifest for gliderlabs/herokuish:latest-20. not found: manifest unknown: manifest unknown.`

I have downloaded the image with 
`docker pull gliderlabs/herokuish:latest-20`
but it changes nothing.

I have also tried buildpacks:set with herokuish 20 and 22 but I still have the error...
I have updated dokku to the last version.

Does anyone know how to make this problem disappear?

**Comment by constantgillet** [comment]
I have solved it by using:

dokku buildpacks:set-property --global stack gliderlabs/herokuish:latest

**Reply by josegonzalez** [reply]
Are you running on a non-amd64 server?

**Reply by constantgillet** [reply]
I am running on a amd64 server
Linux my-domain5.10.0-26-cloud-amd64 #1 SMP Debian 5.10.197-1 (2023-09-29) x86_64 GNU/Linux

**Reply by josegonzalez** [reply]
Oh it looks like you set the image to `gliderlabs/herokuish:latest-20.` previously, with a trailing period. You can see that in the log message. Swapping the stack appeared to have made it work only because you stopped using the trailing period.

**Reply by constantgillet** [reply]
Oh okey thank you :)
